modules/admin_manual/pages/maintenance/upgrading/upgrade_php.adoc

```diff
@@ -1,127 +1,45 @@
-= Upgrade PHP on RedHat 7 and Centos 7
+= Upgrade PHP on RedHat 7 and CentOS 7
 :toc: right
+:toclevels: 1
+:keywords: upgrade, red hat, centos
+:description: Upgrade PHP to versions 7.0 - 7.4 on Red Hat and CentOS so that you can make the most out of your ownCloud installations.
 
 == Introduction
 
-You should, almost, always upgrade to the latest version of PHP, if and
-where possible. And if you’re on a version of PHP older than 5.6 you
-need to upgrade. This guide steps you through upgrading your
-installation of PHP to version 5.6 or 7.0 if you’re on RedHat or Centos
-7.
+You should, almost, always upgrade to the latest version of PHP, if and where possible. 
```

Comment:
```suggestion
You should, almost, always upgrade to the latest version of PHP supported by ownCloud, if and where possible. 
```
(because, e.g., the current latest PHP is 7.4, but that is not yet supported by ownCloud

Reply:
Thanks for adding this one. Really helps to avoid any confusion.